static/css/business.less

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+   cfgov-refresh
+   Business with us pages at /doing-business-with-us/.
+   ========================================================================== */
+
+// Used to set padding to 30px
+@padding: unit(@grid_gutter-width / @base-font-size-px, em);
```

Comment:
The concern is the assumption that you're dividing by `@base-font-size-px`. If the calculation is done in a context where the font size is not `@base-font-size-px` (not that likely a scenario, I grant you, but I'm sure I've seen it), then `@grid_gutter-width__em` would be returning an em value that, in that context, doesn't produce a 30px result.

Good question about variable name with units consistency. I don't have a good answer for that.